runtime/jcl/common/java_dyn_methodhandle.c

```diff
@@ -788,26 +788,19 @@ Java_java_lang_invoke_MethodHandle_vmRefFieldOffset(JNIEnv *env, jclass clazz, j
 	return (jint) J9VMJAVALANGCLASS_VMREF_OFFSET(((J9VMThread *) env));
 }
 
-#ifdef J9VM_OPT_PANAMA
-jlong JNICALL
-Java_java_lang_invoke_MethodHandles_findNativeAddress(JNIEnv *env, jclass jlClass, jstring methodName)
+jobject JNICALL
+Java_java_lang_invoke_MethodHandle_invokeExact(JNIEnv *env, jclass ignored, jobject handle, jobject args)
 {
-	const char *nativeMethodName = NULL;
-	UDATA handle = 0;
-	jlong func = 0;
-	PORT_ACCESS_FROM_ENV(env);
-
-	nativeMethodName = (*env)->GetStringUTFChars(env, methodName, NULL);
-
-	if(0 == j9sl_open_shared_library(NULL, &handle, FALSE)) {
-		if(0 != j9sl_lookup_name(handle, (char *)nativeMethodName, (UDATA *)&func, "")){
-			func = 0;
-		}
-	}
+	throwNewUnsupportedOperationException(env);
+	return NULL;
+}
 
-	return func;
+jobject JNICALL
+Java_java_lang_invoke_MethodHandle_invoke(JNIEnv *env, jclass ignored, jobject handle, jobject args)
+{
+	throwNewUnsupportedOperationException(env);
+	return NULL;
 }
```

Comment:
We don't want these functions here (they have been moved to `java_lang_invoke_MethodHandleNatives.cpp`).

Reply:
@keithc-ca This issue has been resolved. Please take look and let me know if you would like some change in it.